packages/vite/src/node/server/index.ts

```diff
@@ -148,7 +148,7 @@ export interface FileSystemServeOptions {
   /**
    *
    * @expiremental
-   * @deprecated use `fsServe.allow` instead
+   * @deprecated use `fs.allow` instead
```

Comment:
Just to check: since `fsServe` was marked as experimental, we can change it to `fs` without leaving the prev option as deprecated.

In that case, we could remove `root` directly instead of deprecating it since no one is using `fs.root`, no?

Reply:
Yes, I did plan to do that in another PR 😅

Reply:
But better to remove it in this one in case someone makes a realease in the middle and we en up with a new fs.root deprecated option that never existed, no? 😬

Reply:
I actually expect to have two quick PRs in one release, so the changelog would be, which I think would be more clear

- rename `fsServe` to `fs`
- deprecated `fs.root` in favor of `fs.allow`